src/hooks/useTransactionStatus.ts

```diff
@@ -1,9 +1,13 @@
+import { WillReplaceContext } from '@/components/transactions/GroupedTxListItems/WillReplaceProvider'
 import { useAppSelector } from '@/store'
 import { PendingStatus, selectPendingTxById } from '@/store/pendingTxsSlice'
 import { isSignableBy } from '@/utils/transaction-guards'
 import { TransactionSummary, TransactionStatus } from '@gnosis.pm/safe-react-gateway-sdk'
+import { useContext } from 'react'
 import useWallet from './wallets/useWallet'
 
+const REPLACEMENT_STATUS = 'Transaction will be replaced'
```

Comment:
I realize it's tempting to just hardcode an exception for this status, but I think we should include it in the STATUS_LABELS and have a proper type for it.

Reply:
I've added `WILL_BE_REPLACED` to the `TxLocalStatus` type as it otherwise maps to `enum`s. Let me know what you think.